exams/pearson/writers_test.py

```diff
@@ -195,13 +195,60 @@ def setUp(self):  # pylint: disable=missing-docstring
         self.cdd_writer = CDDWriter()
         super().setUp()
 
+    def test_first_name(self):  # pylint: disable=missing-docstring,no-self-use
```

Comment:
We usually put docstrings on all our tests. Also, I think you could refactor these into two test methods and throw `ddt` at it with `name`, `romanized`, `expected` values so you don't have to write so many docstrings. :-)